scheduler/plugin/host.go

```diff
@@ -172,6 +172,15 @@ type stack struct {
 
 	// resultNormalizedScoreList is returned by guest.normalizedscoreFn
 	resultNormalizedScoreList framework.NodeScoreList
+
+	// podInfoToAdd is used by guest.addpodFn
+	podInfoToAdd *v1.Pod
+
+	// podInfoToAdd is used by guest.addpodFn
+	podInfoToRemove *v1.Pod
```

Comment:
```suggestion
	// podInfo is used by guest.addpodFn and guest.removepodFn
	podInfo *v1.Pod
```

You don't need to define same thing again.
Ideally we would like to reuse pod *v1.Pod.
However we can't reuse it because we already use pod as podToSchedule.
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/b32c9665273320841c76eddd75876b51d856a706/scheduler/plugin/host.go#L149

Reply:
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/b32c9665273320841c76eddd75876b51d856a706/scheduler/plugin/host.go#L212-L218
Also you need to define the function like k8sApiPodFn.